files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -102,20 +105,31 @@ function Invoke-RepositoryBuild(
 
         $msBuildArguments | Out-File -Encoding ASCII -FilePath $msBuildResponseFile
 
+        if ($env:KOREBUILD_TEAMCITY_LOGGER) {
+            @"
+/noconsolelogger
+/verbosity:normal
+"/logger:TeamCity.MSBuild.Logger.TeamCityMSBuildLogger,${env:KOREBUILD_TEAMCITY_LOGGER};teamcity"
```

Comment:
Is the plan to manually put the binary on individual agents?

Reply:
Put on agents, yes. Manually, no. TC allows uploading arbitrary .zip files that are automatically installed to all agents. I already set this up so all configs have this:

![image](https://user-images.githubusercontent.com/2696087/35007398-7000796a-faae-11e7-8dbb-e1791cebe4f1.png)